app/src/main/res/raw/menu.json

```diff
@@ -22,6 +22,31 @@
 			}
 		},
 		{
+			"version": 2017072400,
+			"url": "https://debconf17.debconf.org/schedule/pentabarf.xml",
+			"title": "DebConf17",
+			"start": "2017-08-05",
+			"end": "2017-08-12",
+			"metadata": {
+				"icon": "https://debconf17.debconf.org/static/img/favicon/favicon-180.ea5f30d24599.png",
```

Comment:
Not sure what the preferred resolution is, but that seems big enough for most cases.